### DISCUSSION TITLE
Weird Wiki behaviour

### DISCUSSION THREAD

**Discussion by rekmus** [community/community#22646] [New to GitHub]
<p>After adding a page to my Wiki, all the pages now suddenly have it attached:</p>
<p><a href="https://github.com/silgy/silgy/wiki" rel="nofollow noopener">https://github.com/silgy/silgy/wiki</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/0584c2a71901272a237d5d2d4576364ed08c828f.png" data-download-href="/uploads/short-url/MOP9Km6iEhukemNtgfx31zXATR.png?dl=1" title="github.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/36708070/181072270-0d9e5389-c3fb-4dac-8e16-4fdac5c48ed4.png" alt="github.png" data-base62-sha1="MOP9Km6iEhukemNtgfx31zXATR" width="690" height="464" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/0584c2a71901272a237d5d2d4576364ed08c828f_2_690x464.png, https://aws1.discourse-cdn.com/github/optimized/1X/0584c2a71901272a237d5d2d4576364ed08c828f_2_1035x696.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/0584c2a71901272a237d5d2d4576364ed08c828f.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/0584c2a71901272a237d5d2d4576364ed08c828f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github.png</span><span class="informations">1287×866 88.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>It looks to me as a bug.

**Comment by mpboom** [comment, marked as answer]
<p>It’s a bit of a bug.</p>
<p>The idea is that if you edit a page called  <strong>_Footer</strong> it appears as a footer on every page.</p>
<p>Your page is called <strong>OUT_HTML_FOOTER</strong> , and I think the system also thinks that is a match for the  <strong>_Footer</strong> page.</p>
<p>I think the best thing to do now is report this to GitHub support.</p>

**Comment by angela-crist** [comment]
<p>Hi <a class="mention" href="/silgy">@silgy</a>,</p>
<p>Thanks for this feedback! I’ve logged this bug. Though I can’t share a timeline for this, I can tell you that it’s been shared with the appropriate teams.</p>
<p>Please let me know if you have any other questions.</p>
<p>Cheers!</p>
